docker_mcp/server.py

```
@@ -225,6 +276,36 @@
                     return result.structured_content or {"success": True, "data": str(result.content)}
                 return result
                 
+            elif action == "cleanup":
+                # Validate required parameters for cleanup action
+                if not host_id:
+                    return {"success": False, "error": "host_id is required for cleanup action"}
+                if not cleanup_type:
+                    return {"success": False, "error": "cleanup_type is required for cleanup action"}
+                if cleanup_type not in ["check", "safe", "moderate", "aggressive"]:
+                    return {"success": False, "error": f"cleanup_type must be one of: check, safe, moderate, aggressive"}
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)